src/java/org/apache/cassandra/tools/Output.java

```diff
@@ -32,4 +32,14 @@ public Output(PrintStream out, PrintStream err)
         this.out = out;
         this.err = err;
     }
+
+    public void info(String msg, Object... args)
```

Comment:
I see that this is used like 

    @Inject
    protected Output logger;

Is this really a _logger_ thought?

In `NodeTool, the field is called

    private final Output output;

Could we somehow harmonize this? I would call it `output` everywhere.

Reply:
Fixed.